upstairs/src/live_repair.rs

```diff
@@ -4572,8 +4526,26 @@ pub mod repair_test {
         // op# | 0 1 2 | deps
         // ----|-------|-----
         //   0 | RpRpRp|
-        //   1 | RpRpRp| 1
-        //   2 | RpRpRp| 0,1
+        //  *1 | RpRpRp|
+        //   2 | RpRpRp|
+        //   3 | RpRpRp|
+        //   --|-------|-----
+        //   4 | RpRpRp|
```

Comment:
I think this test is not something we would support.
A repair has to either work, or blow up and take the downstairs with it.
haveing a repair come after a repair means something has gone wrong :)

I might even argue that if our repair job finds a previous repair job on it's dependency
list, it should panic.  Though, I don't think the information exists to make that determination.

Reply:
Okay, I removed this test in b85c8755b01e87b3c584c380311d63afe8503c01

It may be possible to check for this condition, but it's not totally obvious, so I'd rather punt on that for now.